_1327/minutes/models.py

```diff
@@ -12,6 +13,31 @@
 MINUTES_VIEW_PERMISSION_NAME = 'view_minutesdocument'
 
 
+class HexColorModelField(models.CharField):
+
+	def __init__(self, *args, **kwargs):
+		kwargs['max_length'] = 7
+		super(HexColorModelField, self).__init__(*args, **kwargs)
+
+	default_validators = [RegexValidator("^#(?:[0-9a-fA-F]{3}){1,2}$")]
+
+
+class MinutesLabel(models.Model):
+	title = models.CharField(max_length=255)
+	color = HexColorModelField()
+
+	def __str__(self):
+		return self.title
+
+	@property
+	def text_color(self):
+		r = int(self.color[1:3], 16)
```

Comment:
in the regex you allow `#fff` as well as `#ffffff`. what happens here if your color does only have 3 digits?